lib/Slackbot_worker.js

```diff
@@ -27,11 +27,11 @@ module.exports = function(botkit, config) {
             botkit.debug('CANNOT SEND WEBHOOK!!');
 
             return cb && cb('No webhook url specified');
-        }
+  }
 
         request.post(bot.config.incoming_webhook.url, function(err, res, body) {
-            if (err) {
-                botkit.debug('WEBHOOK ERROR',err);
+          if (err) {
```

Comment:
Thought it should be a separate pull request for good bookkeeping, but I can merge it into this one if you prefer. Your call.